linkerd/app/outbound/src/lib.rs

```diff
@@ -56,400 +59,481 @@ pub struct Config {
+        prevent_loop: impl Into<PreventLoop>,
+        refine: R,
+        tcp_connect: C,
+        http_router: H,
+        metrics: ProxyMetrics,
+        span_sink: Option<mpsc::Sender<oc::Span>>,
+        drain: drain::Watch,
+    ) -> Pin<Box<dyn Future<Output = Result<(), Error>> + 'static>>
+    where
+        R: tower::Service<dns::Name, Error = Error, Response = dns::Name>
+            + Unpin
+            + Clone
+            + Send
+            + 'static,
+        R::Future: Unpin + Send,
+        C: tower::Service<TcpEndpoint, Error = Error> + Unpin + Clone + Send + Sync + 'static,
+        C::Response: tokio::io::AsyncRead + tokio::io::AsyncWrite + Unpin + Send + 'static,
+        C::Future: Unpin + Send,
+        H: tower::Service<Target<HttpEndpoint>, Error = Error, Response = S>
+            + Unpin
+            + Send
+            + Clone
+            + 'static,
+        H::Future: Unpin + Send,
+        S: tower::Service<
+                http::Request<http::boxed::Payload>,
+                Response = http::Response<http::boxed::Payload>,
+                Error = Error,
+            > + Send
+            + 'static,
+        S::Future: Send,
+    {
+        let Config {
+            canonicalize_timeout,
+            proxy:
+                ProxyConfig {
+                    server: ServerConfig { h2_settings, .. },
+                    disable_protocol_detection_for_ports,
+                    dispatch_timeout,
+                    max_in_flight_requests,
+                    detect_protocol_timeout,
+                    ..
+                },
+        } = self;
 
-        Ok(Outbound { listen_addr, serve })
+        // The stack is served lazily since caching layers spawn tasks from
+        // their constructor. This helps to ensure that tasks are spawned on the
+        // same runtime as the proxy.
+        // Forwards TCP streams that cannot be decoded as HTTP.
+        let tcp_forward = svc::stack(tcp_connect)
+            .push(admit::AdmitLayer::new(prevent_loop.into()))
+            .push_map_target(|meta: tls::accept::Meta| TcpEndpoint::from(meta.addrs.target_addr()))
+            .push(svc::layer::mk(tcp::Forward::new));
+
+        let http_admit_request = svc::layers()
+            // Limits the number of in-flight requests.
+            .push_concurrency_limit(max_in_flight_requests)
+            // Eagerly fail requests when the proxy is out of capacity for a
+            // dispatch_timeout.
+            .push_failfast(dispatch_timeout)
+            //.push(metrics.http_errors.clone())
+            // Synthesizes responses for proxy errors.
+            .push(errors::layer())
+            // Initiates OpenCensus tracing.
+            .push(TraceContextLayer::new(span_sink.clone().map(|span_sink| {
+                SpanConverter::server(span_sink, trace_labels())
+            })));
+        // // Tracks proxy handletime.
+        // //.push(metrics.clone().http_handle_time.layer());
```

Comment:
Handle time has already been updated to 0.2 (in #542), so I don't thinl this needs to be commented out?